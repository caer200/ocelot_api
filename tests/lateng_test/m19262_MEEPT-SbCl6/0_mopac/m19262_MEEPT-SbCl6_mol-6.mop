CHARGES

 
H	5.942466 +1 11.590628 +1 2.215831 +1
C	5.639351 +1 10.990814 +1 2.887281 +1
C	6.197280 +1 11.060299 +1 4.184776 +1
C	4.668902 +1 10.072944 +1 2.581982 +1
C	5.750779 +1 10.097746 +1 5.129562 +1
N	7.139833 +1 12.027379 +1 4.507276 +1
H	4.296599 +1 10.059717 +1 1.708146 +1
C	4.216869 +1 9.152713 +1 3.538443 +1
C	4.770057 +1 9.162358 +1 4.791949 +1
S	6.300884 +1 10.016057 +1 6.762963 +1
C	7.801524 +1 12.104147 +1 5.738165 +1
C	7.313138 +1 13.177127 +1 3.572591 +1
H	3.534267 +1 8.529160 +1 3.319599 +1
H	4.483324 +1 8.527073 +1 5.437665 +1
C	7.513076 +1 11.236275 +1 6.811688 +1
C	8.802003 +1 13.070242 +1 5.943333 +1
H	6.491560 +1 13.266414 +1 3.027797 +1
H	7.408160 +1 14.005237 +1 4.106852 +1
C	8.501701 +1 13.068668 +1 2.637551 +1
C	8.167418 +1 11.369536 +1 8.044340 +1
H	9.044418 +1 13.650982 +1 5.231432 +1
C	9.437378 +1 13.187953 +1 7.161067 +1
H	9.320182 +1 12.876315 +1 3.159981 +1
H	8.632356 +1 13.928036 +1 2.163849 +1
O	8.282797 +1 12.034662 +1 1.696842 +1
H	7.953864 +1 10.781060 +1 8.759042 +1
C	9.112578 +1 12.343702 +1 8.223197 +1
H	10.105874 +1 13.853059 +1 7.278503 +1
C	9.327531 +1 12.009073 +1 0.722882 +1
H	9.543930 +1 12.443953 +1 9.063636 +1
H	9.433253 +1 12.906687 +1 0.318694 +1
H	10.183652 +1 11.760088 +1 1.153921 +1
C	8.983765 +1 11.011482 +1 -0.337304 +1
H	9.593828 +1 11.118899 +1 -1.109201 +1
H	8.056970 +1 11.163247 +1 -0.651478 +1
O	9.102463 +1 9.699734 +1 0.189673 +1
C	8.599694 +1 8.722615 +1 -0.713314 +1
H	8.713035 +1 7.830000 +1 -0.324948 +1
H	7.648133 +1 8.888907 +1 -0.877910 +1
H	9.091794 +1 8.775723 +1 -1.559146 +1
