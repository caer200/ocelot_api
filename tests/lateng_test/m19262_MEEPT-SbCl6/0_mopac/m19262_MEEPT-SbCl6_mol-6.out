 *******************************************************************************
 ** Site#: 29200        For non-commercial use only    Version 19.090L 64BITS **
 *******************************************************************************
 ** Cite this program as: MOPAC2016, Version: 19.090L, James J. P. Stewart,   **
 ** Stewart Computational Chemistry, web: HTTP://OpenMOPAC.net. Days left: 175**
 *******************************************************************************
 **                                                                           **
 **                                MOPAC2016                                  **
 **                                                                           **
 *******************************************************************************

                              PM7 CALCULATION RESULTS

 *******************************************************************************
 *  CALCULATION DONE:                                Mon Oct  7 22:31:21 2019  *
 *  CHARGES    - IDENTIFY AND PRINT CHARGED ATOMS
 *  T=         - A TIME OF 172800.0 SECONDS REQUESTED
 *  DUMP=N     - RESTART FILE WRITTEN EVERY  7200.000 SECONDS
 *******************************************************************************
CHARGES


   ATOM   CHEMICAL          X               Y               Z
  NUMBER   SYMBOL      (ANGSTROMS)     (ANGSTROMS)     (ANGSTROMS)
 
     1       H          5.94246600  *  11.59062800  *   2.21583100  *
     2       C          5.63935100  *  10.99081400  *   2.88728100  *
     3       C          6.19728000  *  11.06029900  *   4.18477600  *
     4       C          4.66890200  *  10.07294400  *   2.58198200  *
     5       C          5.75077900  *  10.09774600  *   5.12956200  *
     6       N          7.13983300  *  12.02737900  *   4.50727600  *
     7       H          4.29659900  *  10.05971700  *   1.70814600  *
     8       C          4.21686900  *   9.15271300  *   3.53844300  *
     9       C          4.77005700  *   9.16235800  *   4.79194900  *
    10       S          6.30088400  *  10.01605700  *   6.76296300  *
    11       C          7.80152400  *  12.10414700  *   5.73816500  *
    12       C          7.31313800  *  13.17712700  *   3.57259100  *
    13       H          3.53426700  *   8.52916000  *   3.31959900  *
    14       H          4.48332400  *   8.52707300  *   5.43766500  *
    15       C          7.51307600  *  11.23627500  *   6.81168800  *
    16       C          8.80200300  *  13.07024200  *   5.94333300  *
    17       H          6.49156000  *  13.26641400  *   3.02779700  *
    18       H          7.40816000  *  14.00523700  *   4.10685200  *
    19       C          8.50170100  *  13.06866800  *   2.63755100  *
    20       C          8.16741800  *  11.36953600  *   8.04434000  *
    21       H          9.04441800  *  13.65098200  *   5.23143200  *
    22       C          9.43737800  *  13.18795300  *   7.16106700  *
    23       H          9.32018200  *  12.87631500  *   3.15998100  *
    24       H          8.63235600  *  13.92803600  *   2.16384900  *
    25       O          8.28279700  *  12.03466200  *   1.69684200  *
    26       H          7.95386400  *  10.78106000  *   8.75904200  *
    27       C          9.11257800  *  12.34370200  *   8.22319700  *
    28       H         10.10587400  *  13.85305900  *   7.27850300  *
    29       C          9.32753100  *  12.00907300  *   0.72288200  *
    30       H          9.54393000  *  12.44395300  *   9.06363600  *
    31       H          9.43325300  *  12.90668700  *   0.31869400  *
    32       H         10.18365200  *  11.76008800  *   1.15392100  *
    33       C          8.98376500  *  11.01148200  *  -0.33730400  *
    34       H          9.59382800  *  11.11889900  *  -1.10920100  *
    35       H          8.05697000  *  11.16324700  *  -0.65147800  *
    36       O          9.10246300  *   9.69973400  *   0.18967300  *
    37       C          8.59969400  *   8.72261500  *  -0.71331400  *
    38       H          8.71303500  *   7.83000000  *  -0.32494800  *
    39       H          7.64813300  *   8.88890700  *  -0.87791000  *
    40       H          9.09179400  *   8.77572300  *  -1.55914600  *
 


          CARTESIAN COORDINATES 

    NO.       ATOM           X           Y           Z

     1         H          5.9425     11.5906      2.2158
     2         C          5.6394     10.9908      2.8873
     3         C          6.1973     11.0603      4.1848
     4         C          4.6689     10.0729      2.5820
     5         C          5.7508     10.0977      5.1296
     6         N          7.1398     12.0274      4.5073
     7         H          4.2966     10.0597      1.7081
     8         C          4.2169      9.1527      3.5384
     9         C          4.7701      9.1624      4.7919
    10         S          6.3009     10.0161      6.7630
    11         C          7.8015     12.1041      5.7382
    12         C          7.3131     13.1771      3.5726
    13         H          3.5343      8.5292      3.3196
    14         H          4.4833      8.5271      5.4377
    15         C          7.5131     11.2363      6.8117
    16         C          8.8020     13.0702      5.9433
    17         H          6.4916     13.2664      3.0278
    18         H          7.4082     14.0052      4.1069
    19         C          8.5017     13.0687      2.6376
    20         C          8.1674     11.3695      8.0443
    21         H          9.0444     13.6510      5.2314
    22         C          9.4374     13.1880      7.1611
    23         H          9.3202     12.8763      3.1600
    24         H          8.6324     13.9280      2.1638
    25         O          8.2828     12.0347      1.6968
    26         H          7.9539     10.7811      8.7590
    27         C          9.1126     12.3437      8.2232
    28         H         10.1059     13.8531      7.2785
    29         C          9.3275     12.0091      0.7229
    30         H          9.5439     12.4440      9.0636
    31         H          9.4333     12.9067      0.3187
    32         H         10.1837     11.7601      1.1539
    33         C          8.9838     11.0115     -0.3373
    34         H          9.5938     11.1189     -1.1092
    35         H          8.0570     11.1632     -0.6515
    36         O          9.1025      9.6997      0.1897
    37         C          8.5997      8.7226     -0.7133
    38         H          8.7130      7.8300     -0.3249
    39         H          7.6481      8.8889     -0.8779
    40         H          9.0918      8.7757     -1.5591

 General Reference for PM7:
 "Optimization of Parameters for Semiempirical Methods VI: More Modifications to the 
 NDDO Approximations and Re-optimization of Parameters", J. J. P. Stewart, J. Mol. Mod., 1:32, 19 (2013)
 http://www.springerlink.com/openurl.asp?genre=article&id=doi:10.1007/s00894-012-1667-x

           Empirical Formula: C17 H19 N O2 S  =    40 atoms



      MOLECULAR POINT GROUP   :   C1  

          COMPUTED CHARGE ON SYSTEM: +0

   COMPUTED CHARGE ON SYSTEM = +0. THIS AGREES WITH THE CHARGE IN THE DATA-SET.

          JOB STOPPED HERE BECAUSE KEYWORD "CHARGES" WAS USED

 ******************************************************************************
 *                                                                            *
 *     Error and normal termination messages reported in this calculation     *
 *                                                                            *
 * JOB STOPPED HERE BECAUSE KEYWORD "CHARGES" WAS USED                        *
 * JOB ENDED NORMALLY                                                         *
 *                                                                            *
 ******************************************************************************



 TOTAL JOB TIME:             0.00 SECONDS

 == MOPAC DONE ==
